### ISSUE TITLE
[Request] Topic Organization and Management

### ISSUE THREAD

**Issue by hotfreshmilk** [lobehub/lobehub#2583]
### 🥰 Feature Description

Currently, LobeChat only allows users to create new chats and view them in a topic list. As the number of chats increases, it becomes difficult to manage and find specific conversations due to the lack of organization.

### 🧐 Proposed Solution

Implement a feature that allows users to create collections or folders for organizing topics. These collections should have customizable names and allow users to easily move completed chats into relevant folders. Users should also be able to create new chats directly within a specific folder.

### 📝 Additional Information

This feature would significantly improve the usability of LobeChat by providing a structured and organized approach to managing conversations. It would allow users to easily categorize and access their chats based on specific topics, making it easier to find and manage information. This feature would be particularly beneficial for users with a large number of chats, as it would prevent them from getting lost in a sea of uncategorized conversations.

**Comment by 9Somboon** [comment]
very good idea.

**Comment by arvinxx** [comment]
You are right. And actually I have a plan to improve it. Please stay tune!

**Comment by arvinxx** [comment]
a small relative improvment: https://github.com/lobehub/lobe-chat/pull/4603, auto group by time. The customization is on the way.

**Comment by schittli** [comment]
> a small relative improvment: [#4603](https://github.com/lobehub/lobe-chat/pull/4603), auto group topics by time. The customization is on the way.

Thank you very much for your trial to improve the handling of hundreds of chats!, 
But **ChatGPT & Co. show how great a grouping function is for individual chats are**. 

It would be great if you could prioritize this feature, because grouping by time is nice, but basically a very poor alternative to real grouping.

Thanks a lot, kind regards,
Thomas
